broker.go

```diff
@@ -68,12 +69,15 @@ func (b *Broker) Open(conf *Config) error {
 	go withRecover(func() {
 		defer b.lock.Unlock()
 
-		dialer := net.Dialer{
-			Timeout:   conf.Net.DialTimeout,
-			KeepAlive: conf.Net.KeepAlive,
+		dialer := &net.Dialer{
```

Comment:
not sure why it did in the first place, changed it back to have the keepalive and use the reference to it in the tls.DialWithDialer call